homeassistant/components/togrill/strings.json

```
@@ -22,6 +22,18 @@
       "failed_to_read_config": "Failed to read config from device"
     }
   },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "active_by_default": "Control active by default"
+        },
+        "data_description": {
+          "active_by_default": "The device only support a single connection at a time. If you intend to use the normal phone application as the normal means of control, this should be disabled. You can connect the device manually using the `Control active` switch entity."
```

Small typo in "support" and we should avoid repeating "normal" in the second sentence:

```suggestion
          "active_by_default": "The device only supports a single connection at a time. If you intend to use the ToGrill phone application as the normal means of control, this should be disabled. You can connect the device manually using the `Control active` switch entity."
```